config_src/drivers/FMS_cap/MOM_surface_forcing_gfdl.F90

```diff
@@ -1787,6 +1796,8 @@ subroutine ice_ocn_bnd_type_chksum(id, timestep, iobt)
   chks = field_chksum( iobt%sw_flux_nir_dif) ; if (root) write(outunit,100) 'iobt%sw_flux_nir_dif', chks
   chks = field_chksum( iobt%lprec          ) ; if (root) write(outunit,100) 'iobt%lprec          ', chks
   chks = field_chksum( iobt%fprec          ) ; if (root) write(outunit,100) 'iobt%fprec          ', chks
+! hfd/Gives error: "Program received signal SIGSEGV: Segmentation fault - invalid memory reference."
+  chks = field_chksum( iobt%seaice_melt    ) ; if (root) write(outunit,100) 'iobt%seaice_melt    ', chks
```

Comment:
I reccomend adding an if associated check here to prevent check summing when not allocated.

Reply:
Done in https://github.com/NOAA-GFDL/MOM6/pull/710/commits/c8dc42ea1484300d3b3a1fa498129b96b93ee382